dpimport/database/__init__.py

```diff
@@ -28,7 +28,8 @@ def connect(self):
             ssl_cert_reqs=ssl.CERT_REQUIRED,
             ssl_certfile=self.config['ssl_certfile'],
             ssl_keyfile=self.config['ssl_keyfile'],
-            ssl_ca_certs=self.config['ssl_ca_certs']
+            ssl_ca_certs=self.config['ssl_ca_certs'],
+            serverSelectionTimeoutMS=300000
```

Comment:
Before merging this PR, undo this. The import failure is due to non-scalable importer:
https://github.com/AMP-SCZ/utility/issues/71#issue-1667077922